README.md

```diff
@@ -46,7 +46,7 @@ Property | Description
 `messageMaxBytes` | Maximum bytes sendable per message including overhead. Default `Sender.messageMaxBytes`
 `messageTimeout` |  Maximum time to wait for messageMaxBytes to accumulate before sending. Default 1 second
 
-#### Don't block the flusher thread
```

Comment:
small adjustment which is unrelated.. it isn't bad to only use one thread and block during sends. It is only bad if you end up blocked to the point where the queue backs up. For example, brave and many others only use 1 sender thread and a blocking sender and they're fine.